test/crawler/keyword/keywords_test.exs

```diff
@@ -0,0 +1,48 @@
+defmodule Crawler.Keyword.KeywordsTest do
+  use Crawler.DataCase, async: true
+
+  alias Crawler.Keyword.Keywords
+  alias Crawler.Keyword.Schemas.Keyword
+
+  describe("list_keywords/1") do
+    test "given a valid user id, returns user's keyword list" do
+      %{id: user_id} = insert(:user)
+      keyword = insert(:keyword, user_id: user_id)
```

Comment:
should we create somemore user's keyword?

Also should we create some keyword that doesn't belongs to the `user` to improve the test quality :D

Reply:
@andyduong1920 Good point! I will add keywords for other users too.

> Also should we create some keyword that doesn't belongs to the user to improve the test quality :D

Notice that, there will not be any keyword that doesn't belong to a user. The user is a must-have.

Reply:
Fixed in [e30ade7](https://github.com/vnntsu/elixir-ic/pull/36/commits/e30ade7853ad15da4e2ca9ca7f92f5a63f2ee6df)